dotnet/Directory.Packages.props

```diff
@@ -63,11 +63,17 @@
     <PackageVersion Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
     <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery" Version="8.2.1" />
     <PackageVersion Include="Microsoft.Orleans.Clustering.Cosmos" Version="8.2.0" />
+    <PackageVersion Include="Microsoft.Orleans.CodeGenerator" Version="8.2.0">
```

Comment:
Make Orlean Version an msbuild property to share across all orleans packages?

Reply:
will do that in a follow-up pr - https://github.com/microsoft/autogen/issues/4152